Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -321,8 +324,7 @@ public IEnumerable<Declaration> Members(Declaration module)
 
         public IEnumerable<Declaration> Members(QualifiedModuleName module)
```

Comment:
This is not coming from this PR, but this is a somewhat confusing misnomer. I think it should better bu called something like `DeclarationInModule`, which is also not completely accurate since the module declarations itself is part of the returned collection.